package.json

```diff
@@ -74,6 +74,8 @@
     "globby": "^9.2.0"
   },
   "devDependencies": {
+    "cordova-android": "^8.0.0",
+    "cordova-ios": "^5.0.0",
```

Comment:
Previous implementation seems to clone master and now it focuses on the current released major.
I suspect this is not an issue in travis or appveyor. What about during a platform release that uses COHO?

The COHO command I suspect is running off master. Will this make a conflict? See blow comment for the conflicting area...

Reply:
This is also missing windows and browser? Any reason to leave off?

Reply:
It is true that previously, the tests would run against the master branch of those two platforms and that with these changes, they would run against the latest version matched by above spec. I think that's a good thing as it makes the tests more stable.

The two platforms included as dependencies are the only ones that have tests written for them in `cordova-js`. So including more platforms here will achieve nothing but a larger download.

Reply:
Ok! stable is what I was expecting behind the change, just confirming that it didn't conflict with COHO and we didn't lose something.

Thank you for clearing that up.